src/main/java/racing/Car.java

```diff
@@ -11,6 +12,7 @@
     private final String name;
 
     private Car(int step, String name) {
+        throwExceptionNullOrLengthGreaterThan(name, 5);
```

Comment:
매직넘버를 의미 있는 이름으로 치환해보면 어떨까요?
```suggestion
        throwExceptionNullOrLengthGreaterThan(name, LIMIT_LENGTH_OF_NAME);
```